### PR TITLE
Few different fixes

### PR DESCRIPTION
**Pull request by iakov**
Fix crash when accel or gyro is not available
Fix strange menu item behavior caused by editablity
Q_ASSERT in release
Playing with threads...

### CODE REVIEW

File: trikGui/accelerometerWidget.cpp [hunk 1]
```diff
@@ -46,7 +46,10 @@ AccelerometerWidget::AccelerometerWidget(trikControl::VectorSensorInterface &acc
 
 void AccelerometerWidget::renew()
 {
-	const auto & value = mAccelerometer.read();
+	const auto &value = mAccelerometer.read();
+	if (value.isEmpty()) {
```
Comment (anastasiia-kornilova) [inline trikGui/accelerometerWidget.cpp]: Please, add logging at least, or better -- some label for user with error description. That is not very obvious UI, we will not understand that gyroscope doesn't work at all.

File: trikNetwork/src/mailboxServer.cpp [hunk 2]
```diff
@@ -124,38 +122,30 @@ void MailboxServer::connect(const QString &ip, int port)
...
-	QMetaObject::invokeMethod(connection, "connect"
-			, Q_ARG(QHostAddress, ip)
-			, Q_ARG(int, port)
-			, Q_ARG(int, mMyPort)
-			, Q_ARG(int, mHullNumber)
-			);
-
-	return connection;
+	const auto c = new MailboxConnection();
+	connectConnection(c);
```
Comment (anastasiia-kornilova) [inline trikNetwork/src/mailboxServer.cpp]: Who will be responsible for deleting it?

Comment (iakov) [reply trikNetwork/src/mailboxServer.cpp]: `startConnection`  steals ownership and transfers it to a worker thread

File: trikGui/accelerometerWidget.cpp (hunk 1 above)
Comment (iakov) [reply trikGui/accelerometerWidget.cpp]: Done. Now there is "Error" literal.

**Pull request merged**